test/test_nn.py

```diff
@@ -1213,30 +1213,15 @@ def test_add_module_raises_error_if_attr_exists(self):
 
     def test_getattr_with_property(self):
```

Comment:
Minor thing: since this test was added with the previous PR and it doesn't test what it's supposed to, maybe we should modify it to correctly check for the right error message and mark it with `@unittest.expectedFailure`? If we implement a workaround in the future, we can verify the test passes then.

Reply:
oh. I didn't know we could do that. I just left a comment saying it raises a wrong AttributeError. I'll change it. I think there's one more place where this happens as well.